org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/DebugAdapter.java

```diff
@@ -964,6 +987,185 @@ public void recyclableThreads(ThreadReference thread) {
             return new Responses.VariablesResponseBody(list);
         }
 
+        Responses.ResponseBody setVariable(Requests.SetVariableArguments arguments) {
+            Map<String, Object> options = new HashMap<>();
```

Comment:
I think the correct way to initialize the "options" is to call VariableFormatter.getDefaultOptions(). Then later we can change its values.